.github/workflows/ci_linux_x64_clang_debug.yml

```diff
@@ -56,9 +60,15 @@ jobs:
             -DCMAKE_BUILD_TYPE=Debug \
             -DIREE_BUILD_PYTHON_BINDINGS=ON \
             -DIREE_ENABLE_LLD=ON \
-            -DIREE_ENABLE_ASSERTIONS=ON
+            -DIREE_ENABLE_ASSERTIONS=ON \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
+            -DCMAKE_C_COMPILER_LAUNCHER=sccache
       - name: CMake - build
-        run: cmake --build ${BUILD_DIR} -- -k 0
+        run: |
+          sccache --zero-stats
+          sccache --show-stats
```

Comment:
We need to `source build_tools/cmake/setup_sccache.sh` here too, so the workflow doesn't fail if used from a context without access to the `AZURE_CCACHE_CONNECTION_STRING` secret.

Reply:
Please address comments before merging in the future.

Reply:
Yeah this was broken: https://github.com/iree-org/iree/actions/runs/10852164410/job/30117574145

Fixing in https://github.com/iree-org/iree/pull/18522